app/src/main/java/com/x8bit/bitwarden/data/platform/manager/BiometricsEncryptionManagerImpl.kt

```diff
@@ -166,7 +221,7 @@ class BiometricsEncryptionManagerImpl(
         )
 
         try {
-            createCipher(userId)
+            createCipherOrNull(userId)
         } catch (e: Exception) {
```

Comment:
@SaintPatrck @david-livefront Would we be comfortable with removing this `try/catch` now? I feel like if we are not ready to remove it, that means we aren't catching enough in `createCipherOrNull`.

Reply:
Removed it and added exception handling where needed.